runtime/onert/backend/cpu/kernel/OperationUtils.h

```diff
@@ -81,7 +81,9 @@ inline nnfw::cker::Shape convertToExtendedCkerShape(const operand::Tensor *tenso
 
 inline nnfw::cker::Shape convertTensorToCkerShape(const operand::Tensor *tensor)
 {
-  assert(tensor);
+  if (tensor == nullptr)
+    return nnfw::cker::Shape();
+
```

Comment:
Is it okay to return empty Shape() where tensor = nullptr? If it could introduce side effect, I will check out of `convertTensorToCkerShape`.

Reply:
I'm not sure that. But It seems to be okay.